.github/workflows/scripts/supported-versions.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Prints supported versions based on the current release branch targeted
+# Version output is in JSON
+
+set -e
+set -x
+
+if git merge-base --is-ancestor origin/release/v5.0 HEAD
+then
+  echo -n "[\"v1.23.9-k3s1\", \"v1.29.3-k3s1\"]"
```

Comment:
If it's available, I would prefer
```
printf '["%s", "%s"]' v1.23.9-k3s1 1.29.3-k3s1
```
etc, for the other two lines. No backslashes, although same number of characters.

Reply:
Sure np, this applies to the other branches as well, so I can make that change to all 3 in  https://github.com/rancher/backup-restore-operator/pull/473, then cherry-pick that commit to the other backports